test/utils/createIdentityProof.testSuite.ts

```diff
@@ -0,0 +1,29 @@
+import { IAM } from "../../src/iam";
+import { createIam, rootOwner } from "../iam.test";
+import { rpcUrl } from "../setup_contracts";
+
+export const createIdentityProofTests = () => {
+    let rootOwnerIam: IAM;
+
+    beforeAll(async () => {
+        rootOwnerIam = await createIam(rootOwner.privateKey, { createDocument: true });
+    });
+
+    //createIdentityProof Test
+    test("Identity token should be defined", async () => {
```

Comment:
What it actually tests?

Reply:
Actually, it is just ensuring that the flow don't break, but I agree with you, the value of this test is very limited (just testing it returns something)